webpack/assets/javascripts/react_app/components/AuditsList/ExpansiveView.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { renderTemplatesDiff } from '../../../foreman_editor';
+
+const showTemplateDiffIfAny = (auditTitle, template) => {
```

Comment:
you can move these functions into `ExpansiveView` class, by that you can use the the props directly rather then pass it through arguments.